docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -51,6 +51,19 @@ storage intact and b) you can build software before you deploy your cluster.
 
 ## Deployment Instructions
 
+> **Warning**: This tutorial uses the following billable components of Google
+> Cloud:
+>
+> - Compute Engine
+> - Filestore
+> - Cloud Storage
+>
+> To avoid continued billing once the tutorial is complete, closely follow the
+> [teardown instructions](#teardown-instructions). It is recomended to deploy
+> this tutorial into a new project that can be deleted when done to avoid left
```

Comment:
nit: "can be deleted when complete, along with all of its resources incurring costs."

Reply:
The suggested wording sounds odd to me. I did my own rewording to make the sentence more direct.